assets/components/TreeItem.vue

```diff
@@ -130,6 +130,12 @@ export default {
         alert('Name must be unique');
         return;
       }
+      // Limit file name size depen
+      if(this.model.name.includes(" ")) {
```

Comment:
It's an interesting approach but I think we should just stick with 14 as the max. The folder icon on the left looks a bit whack and generally split over multiple lines looks awkward imo. Unless you are really passionate about this approach?

Reply:
Yeah no problem, I thought It would solve a potential issue but it probably causes more. I'll change it back